lib/ui/classical_components/barcode_custom_ui.dart

```diff
@@ -44,6 +44,7 @@ class _BarcodeScannerWidgetState extends State<BarcodeScannerWidget> {
 
         /// Use update function to show result overlay on top of the camera or
         //resultStream.add(scanningResult);
+        //liveDetector.resumeDetection(); // resume detection for next snap
```

Comment:
Shouldn't we remove all commented code?

Reply:
no no its examples. we have 3 way of processing result:

- Show result on the overlay on the same screen
- Show result with navigation to preview screen
- Return result to the caller screen and popBack
Commented code its just an examples